api/src/use_cases/get_config/use_case.rs

```diff
@@ -66,32 +68,38 @@ mod unit_tests {
             sunday: None,
         }
     }
+    pub fn config_test_data() -> Config {
+        Config {
+            id: "id".to_string(),
+            business_week: business_week_test_data(),
+        }
+    }
 
-    #[rstest(business_week, expected_business_times,
-        case(test_data(), test_data().into()),
-        case(BusinessWeek::default(), BusinessWeek::default().into()),
+    #[rstest(config, expected_config,
+        case(config_test_data(), config_test_data().into()),
+        case(Config::default(), Config::default().into()),
     )]
     #[tokio::test]
-    async fn should_get_config(
-        business_week: BusinessWeek,
-        expected_business_times: models::BusinessWeek,
-    ) {
+    async fn should_get_config(config: Config, expected_config: models::Config) {
```

Comment:
Do we have a test case to return the default config if no config is present in the DynamoDB backend?

Reply:
If I understood you right, if the id to retrieve the config from dynamodb doesn't exist it doesn't return config.

the tests in DynamoDB shows some use cases for example should_not_get_config_if_no_config_exists, should_save_and_get_config and should_not_save_config_if_it_already_exists

Reply:
I think we should add a test case here for the fallback logic because it is in that same unit. :-)

This is the part that should be covered by a unit test:
```
.unwrap_or_else(|| {
    // Handle the None case here and return a default value or handle the error gracefully
    println!("Error: Config not found. Returning default config.");
    types::Config::default() // Return default config
});
```

Reply:
test is added